templates/landscape/install.html

```diff
@@ -174,257 +182,605 @@ <h2 class="p-heading--5 u-sv3">Why Juju for Landscape</h2>
+        <div class="p-stepped-list--grid">
+          <div class="col-3 col-medium-1 u-hide--small">
+            <p class="p-step-counter"></p>
+          </div>
+          <div class="col-9 col-medium-5">
+            <div class="p-strip is-shallow u-no-padding--top">
+              <h3 class="p-heading--2">Install</h3>
+            </div>
+            <div class="p-strip u-no-padding--top">
+              <ol class="p-list--divided">
+                <li class="p-list__item">
+                  <p class="u-no-padding--top">Deploy the charm</p>
+                  <div class="p-code-snippet">
+                    <pre class="p-code-snippet__block is-wrapped"><code>juju deploy landscape-client --config account-name='standalone' --config --tags='' --config script-users='nobody,landscape,root' --config ping-url="http://${LANDSCAPE_FQDN}/ping" --config url="https://${LANDSCAPE_FQDN}/message-system"</code></pre>
+                  </div>
+                </li>
+              </ol>
             </div>
-            <div class="col-9 col-medium-5">
-              <p class="p-heading--5">Get and install your certificate</p>
-              <div class="p-code-snippet">
-                <pre class="p-code-snippet__block is-wrapped"><code>sudo certbot --apache</code></pre>
-              </div>
+          </div>
+          <hr class="u-no-margin--bottom">
+          <div class="col-3 col-medium-1 u-hide--small">
+            <p class="p-step-counter"></p>
+          </div>
+          <div class="col-9 col-medium-5">
+            <div class="p-strip is-shallow u-no-padding--top">
+              <h3 class="p-heading--2">Configure</h3>
+            </div>
+            <div class="p-strip u-no-padding--top">
+              <ol class="p-list--divided">
+                <li class="p-list__item">
+                  <p class="u-no-padding--top">Relate the charm</p>
+                  <div class="p-code-snippet">
+                    <pre class="p-code-snippet__block is-wrapped"><code>juju relate landscape-client &lt;charm-name&gt;</code></pre>
+                  </div>
+                </li>
+              </ol>
             </div>
           </div>
         </div>
+      </div>
+      
+      <div class="p-tabs__content" tabindex="0" role="tabpanel" id="cloud-init-tab" aria-labelledby="cloud" style="scroll-margin-top:86px;">
+        <div class="p-stepped-list--grid">
+          <div class="col-3 col-medium-1 u-hide--small">
+            <p class="p-step-counter"></p>
+          </div>
+          <div class="col-9 col-medium-5">
+            <p class="p-heading--5">Define variables using Jinja</p>
+            <div class="p-code-snippet">
+              <pre class="p-code-snippet__block is-wrapped"><code># LANDSCAPE_ACCOUNT_NAME: self-hosted Landscape users can leave this as `standalone`
+# Landscape SaaS users can find the account name here: https://landscape.canonical.com/settings
+{% set LANDSCAPE_ACCOUNT_NAME = 'standalone' %}
```

Comment:
```suggestion
{% raw %}
{% set LANDSCAPE_ACCOUNT_NAME = 'standalone' %}
{% endraw %}
```
You need to wrap any jinja syntax in a {% raw %} block to stop it being interpreted as jinja code